src/components/add-site.tsx

```diff
@@ -113,29 +142,36 @@ export default function AddSite( { className }: AddSiteProps ) {
 					focusOnMount="firstContentElement"
 					onRequestClose={ closeModal }
 				>
-					<SiteForm
-						siteName={ siteName || '' }
-						setSiteName={ handleSiteNameChange }
-						sitePath={ sitePath }
-						onSelectPath={ handlePathSelectorClick }
-						error={ error }
-						onSubmit={ handleSubmit }
-						doesPathContainWordPress={ doesPathContainWordPress }
-					>
-						<div className="flex flex-row justify-end gap-x-5 mt-6">
-							<Button onClick={ closeModal } disabled={ isSiteAdding } variant="tertiary">
-								{ __( 'Cancel' ) }
-							</Button>
-							<Button
-								type="submit"
-								variant="primary"
-								isBusy={ isSiteAdding }
-								disabled={ isSiteAdding || !! error || ! siteName?.trim() }
-							>
-								{ __( 'Add site' ) }
-							</Button>
-						</div>
-					</SiteForm>
+					<div ref={ dropRef }>
+						{ isDraggingOver && <DragAndDropOverlay /> }
+						<SiteForm
+							siteName={ siteName || '' }
+							setSiteName={ handleSiteNameChange }
+							sitePath={ sitePath }
+							onSelectPath={ handlePathSelectorClick }
+							error={ error }
+							onSubmit={ handleSubmit }
+							doesPathContainWordPress={ doesPathContainWordPress }
+							fileForImport={ fileForImport }
+							setFileForImport={ setFileForImport }
+							onFileSelected={ handleImportFile }
```

Comment:
Following the comment above, we could use directly `setFileForImport`:

```suggestion
							onFileSelected={ setFileForImport }
```

Reply:
See the comment in https://github.com/Automattic/studio/pull/378#discussion_r1691466673 - I will mark this bit as resolved for now based on my explanation.